FreeRTOS/Source/portable/GCC/RISC-V/port.c

```diff
@@ -196,7 +196,8 @@ void *pvReturnSealer;
 		               __CHERI_CAP_PERMISSION_PERMIT_LOAD_CAPABILITY__);
 
 	pvReturnSealer = cheri_setaddress( pvAlmightyDataCap, SANDBOX_RETURN_OTYPE );
-	pxPortSandboxReturnFunc = cheri_setaddress( pvAlmightyCodeCap, xPortSandboxReturn );
+	pxPortSandboxReturnFunc = cheri_setaddress( pvAlmightyCodeCap, (ptraddr_t)xPortSandboxReturn );

```

Comment:
FreeRTOS style would be the awful:
```suggestion
	pxPortSandboxReturnFunc = cheri_setaddress( pvAlmightyCodeCap, ( ptraddr_t ) xPortSandboxReturn );

```